pkgs/games/gamehub/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv ,fetchFromGitHub ,
+  meson ,ninja ,vala ,pkg-config ,desktop-file-utils ,
+  glib ,gtk3 ,libgee ,libsoup ,json-glib ,sqlite ,webkitgtk ,libmanette ,libXtst
+}:
+stdenv.mkDerivation rec {
+  pname = "GameHub";
+  version = "0.16.0-1";
+  src = fetchFromGitHub {
+    owner = "tkashkin";
+    repo = pname;
+    rev = "${version}-master";
+    sha256 = "sha256-QcpqGydNkmAPrPxp/52+D6MS1I7k+CKHxr3/tpCukP8=";
+  };
+  nativeBuildInputs = [
+    meson ninja vala pkg-config desktop-file-utils
```

Comment:
The app starts for me when I add `wrapGAppsHook` here (even without `appmenu-gtk-module`).

Reply:
With `wrapGAppsHook` added to nativeBuildInputs I get the following error on run:
```
~/.nix-profile/bin/gamehub: /nix/store/9df65igwjmf2wbw0gbrrgair6piqjgmi-glibc-2.31/lib/libc.so.6: version `GLIBC_2.32' not found (required by /nix/store/fdbsckqrhfmwy73mck958pcw5z51wa3b-systemd-247.2/lib/libsystemd.so.0)
```

Reply:
Looks like it's looking for the wrong version of `glibc`. The version in the fork is [2.31](https://github.com/pasqui23/nixpkgs/blob/gamehub/pkgs/development/libraries/glibc/common.nix#L44), but unstable has been on [2.32](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/libraries/glibc/common.nix#L44) for a while, which would explain why it doesn't work (assuming you're on unstable, anyway). Could you try rebasing your branch on master?

Reply:
Yoou're right